pom.xml was modified

git-user-cpp opened the pull request.

git-user-cpp committed:
pom.xml was modified

git-user-cpp committed:
deleted models

git-user-cpp committed:
file StartClass.java was cleaned

git-user-cpp committed:
in class TableHolder:
- in function addRow an exception was re-implemented

git-user-cpp committed:
Update

git-user-cpp committed:
Bug was fixed in TableHolder:
- in function generateUniqueId in loop when
you was calling this.rows... the rows == NULL
so 'rows' were initialized in TableHolder constructor

git-user-cpp committed:
In TableHolder:
- in addRow:
    - added returnUniqueId() call

- method addRow was modified to return a String value
- method returnUniqueId was implemented

git-user-cpp committed:
In TableHolder:
- method verifyId was implemented
- method removeRow was implemented

git-user-cpp committed:
In TableHolder method replaceRow was implemented

git-user-cpp committed:
In TableHolder:
- basic validation was implemented

VladimirPetrat commented on MainModule/src/main/java/model/data/TableHolder.java:
```diff
@@ -15,25 +15,59 @@ public class TableHolder {
         }
         rows.put(generateUniqueId(), rowDataPackage);
+
+        return returnUniqueId();
```
Bug

VladimirPetrat commented on MainModule/src/main/java/model/data/TableHolder.java:
```diff
@@ -15,25 +15,59 @@ public class TableHolder {
+        return returnUniqueId();
+    }
+
+    public void removeRow(String id) throws IllegalAccessException {
```
Why method throws exception up?

git-user-cpp committed:
In class TableHolder:
- method rowIsEmpty was implemented

git-user-cpp committed:
In class TableHolder a bug was fixed

git-user-cpp committed:
In class TableHolder validation was optimized

git-user-cpp committed:
update

VladimirPetrat committed:
Added logic

VladimirPetrat committed:
Added generic to table

git-user-cpp committed:
In class Table:
- private method verifyIdAndColumns was
implemented for code optimization

git-user-cpp committed:
update

git-user-cpp committed:
Fix is required

VladimirPetrat committed:
Update

git-user-cpp committed:
merging

git-user-cpp committed:
merging

git-user-cpp committed:
update

git-user-cpp committed:
In class Table:
- method verifyColumnNamesCorrect was re-implemented

git-user-cpp committed:
In class Table:
- method verifyRows was re-implemented

git-user-cpp committed:
In class Table:
- method updateRowFieldValues was re-implemented

git-user-cpp committed:
Update

git-user-cpp committed:
In class Table:
- method verifyField was added
- method verifyIdAndField was implemented
- additional verification was implemented

VladimirPetrat commented on MainModule/src/main/java/model/data/Table.java:
```diff
@@ -0,0 +1,128 @@
+                .getValue();
+    }
+
+    public DataObject getDataForFieldName(String Id, String fieldName) {
```
Same method as previous. Re-do or drop.

git-user-cpp replied:
My bad) Done

git-user-cpp committed:
In class Table code was optimized

git-user-cpp committed:
In class Table:
- method returnRow was implemented

git-user-cpp committed:
Update
